docs/guides/medical.md

```diff
@@ -177,6 +177,12 @@ Wykaz czynności, które należy zastosować w zależności od ciśnienia i tęt
 
 Należy zapamiętać najważniejszy próg ciśnienia oraz tętna – 50/60, tętno 40. Od tych wartości zaczyna istnieć jakakolwiek szansa na odzyskanie przytomności.
 
+### Zaszywanie ran
+
+Aby zaszyć rany, musisz być medykiem oraz posiadać zestaw do szycia. Upewnij się, że wszystkie rany są zabandażowane i przystąp do zaszywania ran. **Ponieważ każdą część ciała musisz zaszyć osobno**, najlepiej jest zająć się ranami najbardziej rozległymi - pozwoli to na mniejszą utratę krwi w przypadku, gdyby bandaże się otworzyły.
```

Comment:
part "najlepiej jest zająć się ranami najbardziej rozległymi - pozwoli to na mniejszą utratę krwi w przypadku, gdyby bandaże się otworzyły." is unnecesary and more complex tutorial is writted later. Probably its better just write  what is needed to stitch somoene

Reply:
Wooo, good to know about such change in ACE.

>  is unnecesary

I'd leave some mentiong that it's best to start with head/torso stitching as you can't simply put a tourniquet on them if wounds open.

> and more complex tutorial is writted later

Given that this seems a recent change, I doubt tutorial covers part-by-part stitching and how it should be done best.